hadoop-hdds/container-service/pom.xml

```diff
@@ -37,6 +37,10 @@ http://maven.apache.org/xsd/maven-4.0.0.xsd">
       <groupId>org.apache.hadoop</groupId>
       <artifactId>hadoop-hdds-server-framework</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.hadoop</groupId>
```

Comment:
I think it is duplicated, as above lines 37-38 already have added this dependency.